### PR TITLE
Add privacy and T&C docs

### PR DESCRIPTION
**Pull request by kirillbobyrev**

### CODE REVIEW

File: docs/terms.md
Comment (ChristianKuehnel) [inline docs/terms.md:4]: Not sure if this email address is reachable from the internet.

File: docs/terms.md
Comment (ChristianKuehnel) [inline docs/terms.md:5]: I would tweak the wording a bit to something like:
"Google donated the source code and cloud resources to the LLVM project, but is not an official Google product."

File: docs/terms.md
Comment (ChristianKuehnel) [inline docs/terms.md:6]: s/process/problems/

**Comment by kirillbobyrev** [comment]
@kadircet ping

File: docs/privacy.md
Comment (kadircet) [inline docs/privacy.md:13]: s/a distributed/an/

File: docs/privacy.md
Comment (kadircet) [inline docs/privacy.md:26]: in addition success/failure we also log number of results

File: docs/privacy.md
Comment (kadircet) [inline docs/privacy.md:11]: it might be worth pointing users at https://github.com/clangd/llvm-remote-index/blob/master/deployment/entry_point.sh and DockerFile in this section, especially when talking about logging. to point out server works in a "redacted logging" mode, in which it only logs information you mentioned + error templates.

File: docs/terms.md
Comment (kadircet) [inline docs/terms.md:4]: i think we should be wearing our "clangd developer" hats rather than "google c++ tooling team" hats when maintaining this service.

donation of source code sounds weird, i'd just drop that and say `Google donates VM instances in GCP to the LLVM Foundation for hosting this service, and is not tied to it in any other way.`

File: docs/terms.md
Comment (kadircet) [inline docs/terms.md:8]: that's a reasonable promise indeed. but 5% of a week is ~8 hours, and there are weekends. maybe change the promise to "it won't stay down for more than 1 workday"? also if possible i would still avoid that in TC, is it certain that we need this in the TC?

File: docs/terms.md
Comment (kadircet) [inline docs/terms.md:29]: not just difficultes with setup, i think we should make it clear it is the official way to get in touch with maintainers. including bugs & hugs :)

File: docs/terms.md
Comment (kirillbobyrev) [reply docs/terms.md:8]: I think this was something we were asked about so may be a good thing to have but I'm not certain.

**Comment by kirillbobyrev** [comment]
I guess we can push this since the LLVM foundation is happy with the docs?

**Review by kadircet — APPROVED** [review]
LGTM!

**Comment by kirillbobyrev** [comment]
Thanks! FWIW I typically squash commits before merging :) My commit history within a PR is... well... not super clean 😞

**Pull request merged**
